### PR TITLE
[WEEX-86][doc] Update refresh component's document.

### PR DESCRIPTION
**Pull request by miomin**

### CODE REVIEW

File: source/references/components/refresh.md [hunk 1]
```diff
@@ -1,222 +1,141 @@
...
+```
+ - Complete example goes [here](http://dotwe.org/vue/26937c1c74022e79608af118b21bfbc7)
+
+## Child Components
 
-Any other components, like the text and img components, can be put inside the refresh component. And there is a special component named loading-indicator used only inside the refresh or the loading components.
+ - Any other components, like the `<text>` and `<image>` components, can be put inside the refresh component.
 
-* loading-indicator is a child component implemented with default animation effect for the refresh component.
+ - `<loading-indicator>`: There is a special Component named loading-indicator used only inside the `<refresh>` or the `<loading>` Components, which implemented with default animation effect for the refresh component.
```
Comment (jondong) [inline source/references/components/refresh.md]: loading-indicator: this is a dedicated component which provides a default refresh animation effect, can only be used inside the `<refresh>` or `<loading>` components.

File: source/references/components/refresh.md [hunk 2]
```diff
@@ -1,222 +1,141 @@
...
-common styles: check out [common styles for components](../common-style.html)
+| Attribute      | Type     | Value            | Default Value     |
+| ------------- | ------ | -------------------------- | ------- |
+| `display` | String | show / hide             | show      |
 
-### Events
+#### `display`
 
-* onloading triggered when loading
+ - `show`：If a `<loading-indicator>` Component is included in the `<refresh>` Component, it will display and start the animation.
```
Comment (jondong) [inline source/references/components/refresh.md]: The loading animation will be started if a loading-indicator is included in the refresh component.

File: source/references/components/refresh.md [hunk 3]
```diff
@@ -1,222 +1,141 @@
...
 
+ - `hide`：Collapse the refresh view. If a `<loading-indicator>` Component is included in the `<refresh>` Component, it will hide and stop the animation.
```
Comment (jondong) [inline source/references/components/refresh.md]: Collapse the refresh view. It also hides the loading-indicator and stops the loading animation if there's a loading-indicator included in the refresh component.

File: source/references/components/refresh.md [hunk 4]
```diff
@@ -1,222 +1,141 @@
...
 
-#### Restrictions
-* refresh/loading does not support remove action,  Weex 0.9 will fix it.
-* refresh/loading despite setting with display='hide', the refresh/loading view will still appear when scrolling due to known issues. it can be fixed with a another display='hide' when the refresh/loading should be hidden.
-* refresh/loading can only be hidden or displayed with an attribute display with value of show or hide. And there should be a statement of display='hide' when display='show' shows up in an event function, or your scroller may not response to user inputs.
+> **Note：** The display and hide of `<refresh>` Components can only be controlled by an attribute named `display` which has the value of show and hide. A `display="show"` statement is always paired with a `display="hide"` statement.
```
Comment (jondong) [inline source/references/components/refresh.md]: The visibility of refresh component can be controlled by display attribute with the value `show` and `hide`. A `display="show"` should always be paired with a `display="hide"` statement.

File: source/references/components/refresh.md [hunk 5]
```diff
@@ -1,222 +1,141 @@
...
 
-#### Example
+ - Simple example :
```
Comment (jondong) [inline source/references/components/refresh.md]: Example:

File: source/references/components/refresh.md [hunk 6]
```diff
@@ -1,222 +1,141 @@
...
-  }
-</style>
 ```
+ - Complete example goes [here](http://dotwe.org/vue/26937c1c74022e79608af118b21bfbc7)
 
-## Refresh Components
+## Styles
 
-To be rendered properly, the refresh/loading Components must appear inside the Scroller Component or the List Component.
+ - Support all common styles, check out [common styles](../common/common-style)
```
Comment (jondong) [inline source/references/components/refresh.md]: please check out the common styles

File: source/references/components/refresh.md [hunk 7]
```diff
@@ -1,222 +1,141 @@
...
 
-### Child Components
+## Events
 
-Any other components, like the text and img components, can be put inside the refresh component. And there is a special component named loading-indicator used only inside the refresh or the loading components.
+### `onrefresh`
 
-* loading-indicator is a child component implemented with default animation effect for the refresh component.
+ - Triggered when the scroller or list has been pulled down.
```
Comment (jondong) [inline source/references/components/refresh.md]: is pulled down.

File: source/references/components/refresh.md [hunk 8]
```diff
@@ -1,222 +1,141 @@
...
 
-### Attributes
-* display has value of show or hide, default value is show.
+### `onpullingdown` <span class="weex-version">v0.6.1+</span>
 
-Other attributes please check out the [common attributes](../common-attrs.html).
+ - Triggered when the scroller or list has been pulled down. you can get `dy`, `pullingDistance`, `viewHeight` and `type` from onpullingdown `event` object :
```
Comment (jondong) [inline source/references/components/refresh.md]: is pulled down.
The attributes `dy`... are accessible from the `event` object:

File: source/references/components/refresh.md [hunk 9]
```diff
@@ -1,222 +1,141 @@
...
+  - `viewHeight` : The height of refreshView
+  - `type` : "pullingdown" constant string type for this event
 
-### Styles
-common styles: check out [common styles for components](../common-style.html)
 
-### Events
-* `onrefresh`: triggered when the scroller has been pulled down
-* `onpullingdown`: triggered when the scroller has been pulled down. you can get `dy, pullingDistance, viewHeight, type` from onpullingdown event object.
+ - Simple example :
```
Comment (jondong) [inline source/references/components/refresh.md]: Example:

File: source/references/components/refresh.md [hunk 10]
```diff
@@ -1,222 +1,141 @@
...
-    text-align: center;
-    color: #41B883;
-  }
-</style>
 ```
+ - Complete example goes [here](http://dotwe.org/vue/26937c1c74022e79608af118b21bfbc7)
+
+## Usage Notes
+
+ - On the Android platform, it is not allowed for the refresh view to be pushed back when it is been pulling down or its onrefresh event is tiggered, while it is allow as for iOS platform and HTML5.
```
Comment (jondong) [inline source/references/components/refresh.md]: There is a restriction on Android platform that the refresh view cannot be pushed back during the pulling down process or the`onrefresh` event is triggered, while it is allowed both on iOS and HTML5.

File: source/references/components/refresh.md [hunk 11]
```diff
@@ -1,222 +1,141 @@
...
 
-Any other components, like the text and img components, can be put inside the refresh component. And there is a special component named loading-indicator used only inside the refresh or the loading components.
+### `onrefresh`
 
-* loading-indicator is a child component implemented with default animation effect for the refresh component.
+ - Triggered when the scroller or list has been pulled down.
 
-### Attributes
-* display has value of show or hide, default value is show.
+### `onpullingdown` <span class="weex-version">v0.6.1+</span>
```
Comment (cxfeng1) [inline source/references/components/refresh.md]: The event name is 'pullingdown', not 'onpullingdown'

**Pull request merged**